docs/mkdocs.yml

```diff
@@ -18,27 +22,30 @@ nav:
   - AnyPath: "anypath-polymorphism.md"
   - Testing code that uses cloudpathlib: "testing_mocked_cloudpathlib.ipynb"
   - Integrations: "integrations.md"
+  - Changelog: "changelog.md"
   - API Reference:
       - CloudPath: "api-reference/cloudpath.md"
-      - S3:
-          - S3Client: "api-reference/s3client.md"
-          - S3Path: "api-reference/s3path.md"
+      - AnyPath: "api-reference/anypath.md"
       - Azure:
           - AzureBlobClient: "api-reference/azblobclient.md"
           - AzureBlobPath: "api-reference/azblobpath.md"
+      - S3:
+          - S3Client: "api-reference/s3client.md"
+          - S3Path: "api-reference/s3path.md"
       - GS:
           - GSClient: "api-reference/gsclient.md"
           - GSPath: "api-reference/gspath.md"
-      - AnyPath: "api-reference/anypath.md"
-      - Local: "api-reference/local.md"
+      - Submodules:
+          - cloudpathlib.client: "api-reference/client.md"
+          - cloudpathlib.exceptions: "api-reference/exceptions.md"
+          - cloudpathlib.local: "api-reference/local.md"
 
 markdown_extensions:
   - pymdownx.highlight
   - pymdownx.superfences
   - toc:
       permalink: True
-  - codehilite:
-      use_pygments: true
```

Comment:
This configuration appears to be the default so I don't think it's necessary and detracts from readability of the config https://squidfunk.github.io/mkdocs-material/reference/code-blocks/#highlight